Description: Fix bashisms in enlightenment_remote
Author: Ross Vandegrift <ross@kallisti.us>
Forwarded: no
Last-Update: 2017-11-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/data/tools/enlightenment_remote
+++ b/data/tools/enlightenment_remote
@@ -8,7 +8,7 @@
 #-------------------------------------------------------------------------------
 show_help(){
    # Actual implemented features:
-echo -e "
+printf "
 OPTIONS:
   -h This help
   -help This help
@@ -64,7 +64,7 @@
 #   Show info about the new implementation of enlightenment_remote
 #-------------------------------------------------------------------------------
 show_help_new_tool(){
-   echo -e "
+   printf "
 This is a new implementation of enlightenment_remote, it uses dbus calls to
 enlightenment so it is using the new system, this tool is a front-end just like
 the original tool, created for the ease-to-use of the users and in order to
@@ -449,7 +449,7 @@
 #       RETURNS:  just finishes and exit
 #===============================================================================
 
-if ! type dbus-send > /dev/null 2>&1; then
+if ! which dbus-send > /dev/null 2>&1; then
    echo "E: dbus-send command not found, please install it first"
    exit 1
 fi
